src/containers/guest-home-page/login-dialog/LoginDialog.js

```diff
@@ -8,7 +8,7 @@ import LoginForm from '~/containers/guest-home-page/login-form/LoginForm'
 import useForm from '~/hooks/use-form'
 import useConfirm from '~/hooks/use-confirm'
 import { ModalContext } from '~/context/modal-context'
-import { email, password } from '~/constants/validation/login'
+import { email, password } from '~/validations/login'
```

Comment:
I think the "validations" folder can be moved to the utils folder.

Reply:
Validations moved to the utils folder